inappmessaging/USERGUIDE.md

```diff
@@ -337,6 +345,9 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 5.0.0 (in-progress)
+* SDKCF-4071: **Breaking Change:** Added new method for providing id tracking identifier in `UserInfoProvider` interface class.
+
```

Comment:
Can we add the functionality without making it a breaking change? e.g. as an optional interface function.

Reply:
It is possible for Kotlin apps. But it will still be required for Java apps to implement the new method.

Reply:
from discussion (with PDM), we will consider this as breaking change and just add a note that it is not "required" for Kotlin apps.
Is this ok?

Reply:
yep, it's ok. Let's make sure that it's a semver major version bump in that case.

Reply:
@donnie-jp 
Updated userguide for this.